docs/production/troubleshooting.md

```diff
@@ -116,6 +116,13 @@ Zulip's services; if you want to restart just one of them, you can use
 supervisorctl restart zulip-django
 ```
 
+:::{warning}
+A configuration file might be used by multiple services, so generally
+`scripts/restart-server` is the correct tool to use for reloading
+purposes. Only use `supervisorctl restart` for an individual service
+if you're confident you understand the impact.
```

Comment:
"understand the impact" reads a little vague; maybe something like "if you're confident you know the service that needs restarting"

Also, do we make a distinction between "reloading" (as you use in the copy here) and "restart," as in the `scripts/restart-server`?

Reply:
Yeah, I think "understand the impact" does nothing to educate folks to what the impact will be, so they will just assume they understand it. :)

Specifying that it should not be done for settings changes (which is probably a frequent cause of restarts) because it will cause inconsistent behaviour is more specific.

I also wonder if we should use a code block for `scripts/restart-server`, instead of an inline code span, so it's more visible?

Reply:
`if you're confident you understand the impact` was meant to convey something like `you should be an advanced user for this`, but yeah perhaps it doesn't succeed at that. 

How about something like I pushed now to try to apply your feedback? Moved `restart-server` to a code block and reworder the explanation somewhat:
![image](https://github.com/zulip/zulip/assets/45007152/8a0971fa-f31f-4128-9d8a-a42853f61286)

> Also, do we make a distinction between "reloading" (as you use in the copy here) and "restart," as in the scripts/restart-server?

Hmm, I don't think so - perhaps Alex may have other thoughts on this.

Reply:
I like the new wording, @mateuszmandera!

We're not consistent about reload vs restart.  In other software, "reload" is generally implied to be less drastic than "restarting," so it's possible that we should standardize on "restart" because we _are_ taking services down and back up again.